lldb/tools/debugserver/source/MacOSX/MachThread.cpp

```diff
@@ -265,6 +264,25 @@ bool MachThread::GetBasicInfo(thread_t thread,
   return false;
 }
 
+struct thread_extended_info *MachThread::GetExtendedInfo() {
+  if (MachThread::GetExtendedInfo(m_mach_port_number, &m_extended_info))
+    return &m_extended_info;
+  return NULL;
+}
+
+bool MachThread::GetExtendedInfo(thread_t thread,
+                                 struct thread_extended_info *basicInfoPtr) {
```

Comment:
It's confusing to have a variable called "basicInfoPtr" actually be an "extended_info"...  Can you change this name?

Reply:
fixed